library/src/main/java/pl/rspective/voucherify/android/view/VoucherCheckoutView.java

```diff
@@ -40,6 +40,7 @@
     private Drawable voucherIcon;
     private Drawable validVoucherIcon;
     private Drawable invalidVoucherIcon;
+    private String invalidVoucherErrorMessage;
```

Comment:
@polok it was designed specifically to show when given code is invalid. I agree that one wanted to show an error message in different cases. So I changed my solution. Now the setter is named `setVoucherErrorMessage` - once it's invoked the message appears. It's simpler and more flexible for a developer. The only disadvantage is that I had to remove the custom attribute since it doens't makes sense to have a static message for various cases.